brave/brave-core
# Moves redeem from common to specific files

Resolves https://github.com/brave/brave-browser/issues/9773

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- claim grant
- send a tip
- make sure that tip goes through

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
